src/main/javascript/src/store/treatment.module.js

```diff
@@ -38,6 +38,30 @@ const actions = {
       console.log('createTreatment catch', error)
     }
   },
+  async duplicateTreatment ({commit}, payload) {
```

Comment:
Maybe we should call this "updateTreatment" instead of "duplicateTreatment", since duplicate is a different endpoint on the backend?